README.md

```diff
@@ -1,41 +1,194 @@
-# reactome_galaxy
-[![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg?style=flat)](http://bioconda.github.io/recipes/reactome-cli/README.html)
+# Reactome Galaxy Integration
 
+A command-line utility and Galaxy platform integration for the Reactome pathway analysis service. This project provides both a standalone Java CLI tool and Galaxy tool wrappers for performing pathway analysis using the Reactome database.
 
-Command-line utility and for running the Reactome analysis service and
-integration into Galaxy platform.
+## Overview
 
-Getting started requires a jdk >= 11 for the cli and conda for the galaxy integration.
-Build the cli and run an sample gene list analysis with the following commands:
+This application enables users to perform pathway analysis using Reactome's comprehensive biological pathway database. It supports multiple types of analysis including gene set analysis, tissue-specific analysis, and species comparison analysis. The tool can be used both as a standalone command-line application and as integrated tools within the Galaxy bioinformatics platform.
+
+### Key Features
+
+- **Multiple Analysis Types**: Supports gene, tissue, and species pathway analysis
+- **Galaxy Platform Ready**: Includes complete Galaxy tool configurations with tests
+- **Comprehensive Input**: Accepts various identifier formats and gene lists
+- **Comprehensive Output**: Generates detailed pathway analysis results in multiple formats
+
+### Analysis Capabilities
+
+1. **Gene Analysis**: Analyze gene lists against Reactome pathways
+2. **Tissue Analysis**: Perform tissue-specific pathway analysis
+3. **Species Analysis**: Compare pathways across different species
+4. **GSA Analysis**: Gene Set Analysis using the ReactomeGSA R package
+
+### CLI Technical Overview
+
+The Reactome CLI is a simple command line interface which wraps calls to the Reactome web apis written in Kotlin. It provides an easy
+entrypoint for workflow engines to call these apis, without needing to worry about constructing input HTTP requests and processing 
+outputs.
+
```

Comment:
@kzuberi do you think you could help me flesh this out a little? Maybe a bit about the software design and choices made?